.terraform.lock.hcl

```diff
@@ -79,6 +81,25 @@ provider "registry.terraform.io/hashicorp/local" {
   ]
 }
 
+provider "registry.terraform.io/hashicorp/random" {
```

Comment:
nitpick: We keep the block here to pin the version but we'll have to do https://github.com/jenkins-infra/aws/issues/221